ppl/archive/import.go

```diff
@@ -15,17 +18,24 @@ import (
 	"github.com/brimsec/zq/zng"
 	"github.com/brimsec/zq/zng/resolver"
 	"go.uber.org/multierr"
+	"golang.org/x/sync/errgroup"
 )
 
 var (
 	// ImportBufSize specifies the max size of the records buffered during import
 	// before they are flushed to disk.
 	ImportBufSize          = int64(sort.MemMaxBytes)
 	ImportStreamRecordsMax = zngio.DefaultStreamRecordsMax
+	// DefaultImportFlushTimeout is the default time that an archive will wait
+	// before flushing to disk and closing a tsdir has not received any new
+	// data.
+	DefaultImportFlushTimeout = time.Second
 )
 
-// For unit testing.
-var importLZ4BlockSize = zngio.DefaultLZ4BlockSize
+var (
+	// For unit testing.
+	importLZ4BlockSize = zngio.DefaultLZ4BlockSize
+)
```

Comment:
Nit: Don't use `var (...)` for a single variable.

Reply:
Yep